pkg/systemd/logs.scss

```diff
@@ -4,11 +4,6 @@
 @import "global-variables.scss";
 @import "@patternfly/patternfly/utilities/Flex/flex.scss";
 
-// https://github.com/patternfly/patternfly-react/issues/5993
-.pf-v5-c-popover.pf-m-width-auto {
-  --pf-v5-c-popover--MaxWidth: min(300px, 90%);
-}
-
```

Comment:
I'm referring to this issue: https://github.com/cockpit-project/cockpit-machines/issues/1794

(So this override doesn't actually fix the problem in that case, but we might need to revisit this problem and perhaps even make another override instead. Machines should probably change the way it calls that popover, but it also shouldn't happen like that by default too.)

Reply:
Ironically, that issue in Machines is _lacking_ this CSS workaround for whatever reason, and when I add it, it fixes the problem.

As-is, without the workaround:

![Screen Shot 2024-09-10 at 11 37 39](https://github.com/user-attachments/assets/3610f123-e983-460e-a6ac-a83803069ef8)


With the workaround:

![Screen Shot 2024-09-10 at 11 36 55](https://github.com/user-attachments/assets/02b5f4a6-8bb5-4c9b-894a-603779fb2829)

Reply:
Uh, so we make this a global override? Imo its not great either

Reply:
I agree that none of these options are great. But this is what PatternFly is giving us.

I'd personally implement this in a different way that wouldn't require so much JS and even so much CSS and DOM to do the same thing, but... (It's obvious how it should be, but this isn't done in that way, therefore it has lots of bugs like this.)

Reply:
The end result is that people using the software would care if they can actually read the text or not. Screenshot 1 is clearly wrong and bad, whereas screenshot 2 is actually functional. Regardless of implementation.

Reply:
But, meanwhile, it would be completely fine to merge this PR without this change. This change might be needed, and may even need to be extended (as seen above).